packages/cactus-plugin-bungee/src/main/typescript/public-api.ts

```diff
@@ -0,0 +1,10 @@
+import { IPluginFactoryOptions } from "@hyperledger/cactus-core-api";
+import { PluginFactoryBUNGEE } from "./plugin-factory-bungee";
+
+export { PluginBUNGEE, IPluginBUNGEEOptions } from "./plugin-bungee";
```

Comment:
@eduv09 Just marking it here too that after the rename please don't forget to rename the imported files/paths, the classes that are being exported as well (it will be annoying to refactor it everywhere but much easier if you use the search function of VSCode wisely. :-)